modules/nf-core/interproscan/tests/main.nf.test

```diff
@@ -8,58 +8,90 @@ nextflow_process {
     tag "modules_nfcore"
     tag "interproscan"
 
-    test("Annotates set of input proteins in an output tsv file") {
+    // test("sarscov2 - proteome_fasta") {
```

Comment:
Can you add a comment saying why they have been commented out.

Reply:
done